08LessonEight/homework/exercise-06.html

```diff
@@ -2,9 +2,33 @@
 <html>
 
 <head>
-    <title>Exercise 06</title>
+  <title>Exercise 06</title>
+  <link href="css/exercise-06-style.css" rel="stylesheet" type="text/css"/>
 </head>
 
-<body></body>
+<body>
+  <div>
+    <div class="division">
+      <img src="img/truck.jpg"/>
+      <h3>SAFE & SECURE DELIVERY</h3>
+      <p>Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.</p>
+      <a href="#">Read More</a>
+    </div>
 

```

Comment:
good structure here, Harry. try commenting a bit more though for future projects.